### PR TITLE
refactor: update copy, add link and remove unfinished language selector

### PR DESCRIPTION
**Pull request by secondl1ght**
### Describe the changes you have made in this PR

Updated copy for language selector in Settings based on user feedback, also added a link to the Weblate for translations, and removed unfinished languages to avoid further confusion. 

### Link this PR to an issue

#1192 

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes (If any)
image

### How has this been tested?
Ran locally

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by escapedcat** [comment]
Would this work to avoid an extra `link`-prop?:
https://stackoverflow.com/a/70769183/1667461

**Comment by secondl1ght** [comment]
> Would this work to avoid an extra `link`-prop?: https://stackoverflow.com/a/70769183/1667461

Can you pass a React component in as a component prop though, I thought you could not? I can try and see.

File: src/app/components/Setting/index.tsx [hunk 1]
```diff
@@ -2,17 +2,18 @@ type Props = {
   title: string;
   subtitle: string;
   children: React.ReactNode;
+  link?: string[];
 };
 
-function Setting({ title, subtitle, children }: Props) {
+function Setting({ title, subtitle, children, link }: Props) {
```
Comment (bumi) [inline src/app/components/Setting/index.tsx:8]: I am unhappy about this. that' is the problem with such abstractions... you loose flexibility. 
If we now add custom code or attributes here and there it increases a lot of the complexity. 
No other Setting needs the link, so I would actually avoid adding this. is that link really that valuable?
and if? can we pass in html as the subtitle? (without removing escaping)

**Review by bumi — CHANGES_REQUESTED** [review]

File: src/app/components/Setting/index.tsx (hunk 1 above)
Comment (bumi) [reply src/app/components/Setting/index.tsx:8]: `Trans` sounds interesting. 

I am also OK with removing the link

Comment (secondl1ght) [reply src/app/components/Setting/index.tsx]: I was able to remove the prop and use the `Trans` comp.

**Comment by secondl1ght** [comment]
Refactored to remove the new prop and use the `Trans` component.

File: src/app/screens/Settings.tsx [hunk 2]
```diff
@@ -99,7 +99,17 @@ function Settings() {
             </p>
           )}
         </Setting>
-        <Setting title={t("language.title")} subtitle={t("language.subtitle")}>
+        <Setting
+          title={t("language.title")}
+          subtitle={
```
Comment (escapedcat) [inline src/app/screens/Settings.tsx:104]: Would be great if we could avoid the extra link prop.  
Have a look at this: https://www.createit.com/blog/i18next-react-links-inside-translations/  
This seems more flexible. We might have these kinda situations more often in the future with multiple links.

Comment (secondl1ght) [reply src/app/screens/Settings.tsx:104]: What do you mean the remove extra link prop? I had a look at that article and im not sure what problem this is solving for what we are doing here?

File: src/i18n/locales/en/translation.json [hunk 3]
```diff
@@ -205,7 +205,8 @@
       },
       "language": {
         "title": "Language",
-        "subtitle": "Alby goes international! help us translate Alby in your language"
+        "subtitle": "Translations are not 100% finished yet.",
```
Comment (escapedcat) [inline src/i18n/locales/en/translation.json:208]:
```js
{
  "gdpr_policy_links": "I agree to <link1>Privacy Policy</link1> and <link2>Terms and Conditions</link2>",
  "gdpr_required_msg": "Please accept Privacy Policy"
}
```

This looks like we wouldn't need `link` and `subtitle` but only `subtitle` if we use this right. 
I hope we can do this:
```js
{
  "subtitle": "Translations are not 100% finished yet. <link>Help us translate Alby into your language!</link>",
}
```

**Comment by bumi** [comment]
I am kinda lost here. what is this `Trans` doing exactly? 
and do we really need that link?

**Comment by escapedcat** [comment]
> I am kinda lost here. what is this `Trans` doing exactly?

[Trans helps us to handle html-tags](https://www.createit.com/blog/i18next-react-links-inside-translations/) within translations strings. We already use it in other places.  
Getting familiar with its usage will help with all further situations where we have to deal with html-tags.

> and do we really need that link?

I believe it's a good starting point for people who might be interested in adding languages.

**Comment by bumi** [comment]
ok, then can this be merged?

**Comment by escapedcat** [comment]
> ok, then can this be merged?

No, we should get rid of the extra `link`-prop

**Review by escapedcat — APPROVED** [review]

File: src/app/components/LocaleSwitcher/LocaleSwitcher.tsx [hunk 4]
```diff
@@ -32,7 +32,6 @@ export default function LocaleSwitcher() {
   return (
     <Select name="locale" value={dropdownLang} onChange={languageHandler}>
       <option value="en">English</option>
-      <option value="hi">हिन्दी</option>
```
Comment (bumi) [inline src/app/components/LocaleSwitcher/LocaleSwitcher.tsx:35]: we actually got many languages now and some are pretty much completely translated (from what can be translated right now)
Ideally we load those even dynamically here

**Review by bumi — COMMENTED** [review]
If we show the current languages in the dropdown, then I am ok

File: src/app/components/LocaleSwitcher/LocaleSwitcher.tsx (hunk 4 above)
Comment (escapedcat) [reply src/app/components/LocaleSwitcher/LocaleSwitcher.tsx:35]: Yeah, possibly. On the other hand a new language would need to wait for a new release or a patch release. Not that hard to manage, right? Let's talk about it.

Comment (bumi) [reply src/app/components/LocaleSwitcher/LocaleSwitcher.tsx:35]: not hard, but right now it's not there. let's add it.

**Pull request merged**